io_scene_godot/converters/material_node_tree/shaders.py

```diff
@@ -205,7 +204,7 @@ def add_image_texture(self, uniform_var, image):
     def to_string(self):
         """Serialization"""
         return '\n'.join([
-            self._shader_type,
+            "shader_type spatial;",  # shader type spatial
```

Comment:
The comment is a bit redundant. I can read that from the code. It's a bit like:
`x += 3  # adds 3 to x`

Reply:
Ah, I will fix it